packages/discord-error/README.md

```diff
@@ -0,0 +1,25 @@
+# discord-error
+
+Creates a JavaScript Error from the body response of a Discord API JSON error message. It is intended to be used primarily in libraries rather than client code.
+
+## Usage
+
+```js
+import DiscordError, { isDiscordError } from "discord-error";
```

Comment:
Default exports 😞 
Please try to avoid them as they are awful for cjs users, require-ing code like `const { default: DiscordError } = require('discord-error')`

Reply:
Made the conscious decision to not support cjs earlier in the project, but I can agree mixing them is ugly. I'll think on it a bit